packages/illustrations/README.md

```diff
@@ -10,18 +10,29 @@ Ultraviolet Illustrations is a set of illustrations.
 $ pnpm add @ultraviolet/illustrations
 ```
 
-### Usage
+## Usage
+### Assets
 
 ```js
 import { instanceOriginal } from '@ultraviolet/illustrations/products/instance'
 
 const App = () => <img src={instanceOriginal} alt="illustration instance" />
 ```
+### Components 
+```js
+import { DynamicIllustration } from '@ultraviolet/illustrations'
 
+const App = () => <DynamicIllustration name="success">
+```
 ## Documentation
 
 Checkout our [documentation website](https://storybook.ultraviolet.scaleway.com/).
 
 ## Contributing
 
 📝 You can participate in the development and [start contributing](/CONTRIBUTING.md) to it.
+
+### Adding an asset
+The whole process is automatic. Simply add the assets you want to include in the package to the correct folder in `assets/` and create a pull request. Once the pull request is approved and the branch is merged, the illustrations will be added to an S3 bucket and exported in the package automatically.
+### Deleting an asset
```

Comment:
You could also mention that on renaming the old image will be replaced and uploaded into a S3 so we cover all the cases (add, remove and edit)